openfermioncirq/trotter/simulate_trotter_test.py

```diff
@@ -35,40 +33,49 @@ def fidelity(state1, state2):
     return abs(numpy.dot(state1, numpy.conjugate(state2)))**2
 
 
+def load_molecular_hamiltonian(
```

Comment:
Why is this function in two different PRs? This is clearly code that does not belong in OpenFermion-Cirq. You can put it in OpenFermion but it does not belong here.

Reply:
Opened https://github.com/quantumlib/OpenFermion/pull/432 for it.